doc/tutorials/launch_uos.sh

```diff
@@ -135,11 +135,10 @@ acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
   -s 6,virtio-hyper_dmabuf \
   -s 8,wdt-i6300esb \
   -s 3,virtio-blk,/data/$5/$5.img \
-  -s 4,virtio-net,$tap \
+  -s 4,virtio-net,tap=$6,mac_seed=$mac_seed \
```

Comment:
If I read the script properly, the tap device created by the script is actually called `tap_$6`, so this is wrong, isn't it?

Reply:
As previous.
The type is tap and the name is the parameter $6.

Reply:
> As previous. The type is tap and the name is the parameter $6.

The script is prepending `tap_` before creating it (or checking if it exists)... look at this line: https://github.com/projectacrn/acrn-hypervisor/blob/2bcdddaf130537796e88390d413076282e4d99cd/doc/tutorials/launch_uos.sh#L46

That means the script is assuming the TAP device is `tap_$6` but the argument to `acrn-dm` uses `%6`, this will not work.